### PR TITLE
Update the documentation for the ls command for the rename to list.

### PR DESCRIPTION
**Pull request by danielmcgraw**
Update to the documentation if/when https://github.com/yarnpkg/yarn/pull/1940 is pulled in to yarn.

### CODE REVIEW

**Comment by danielmcgraw** [comment]
@Daniel15 Fixed that up. Thanks for letting me know.

File: _data/guides.yml [hunk 1]
```diff
@@ -81,9 +81,9 @@
   - id: docs_cli_logout
     path: /docs/cli/logout
     tags: ["cli-logout"]
-  - id: docs_cli_ls
-    path: /docs/cli/ls
-    tags: ["cli-ls"]
+  - id: docs_cli_list
+    path: /docs/cli/list
+    tags: ["cli-list"]
```
Comment (charlierudolph) [inline _data/guides.yml]: This should reordered to be above login. logout in order to keep the commands in alpha order

File: _data/i18n/en.yml [hunk 2]
```diff
@@ -125,7 +125,7 @@ docs_cli_licenses: yarn licenses
 docs_cli_link: yarn link
 docs_cli_login: yarn login
 docs_cli_logout: yarn logout
-docs_cli_ls: yarn ls
+docs_cli_list: yarn list
```
Comment (charlierudolph) [inline _data/i18n/en.yml]: This should be moved above login / logout to retain alpha order.

**Comment by namhong2001** [comment]
I also encountered mismatch between docs and `yarn ls` errors. I hope this to be merged soon.

File: en/docs/cli/global.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ layout: guide
 
 ##### `yarn global` <a class="toc" id="toc-yarn-global" href="#toc-yarn-global"></a>
 
-`yarn global` is a prefix used for a number of commands like `add`, `bin`, `ls` and `remove`. They behave identically to their normal versions except that they use a global directory to store packages. The `global` command makes binaries available to use on your operating system.
+`yarn global` is a prefix used for a number of commands like `add`, `bin`, `list` and `remove`. They behave identically to their normal versions except that they use a global directory to store packages. The `global` command makes binaries available to use on your operating system.
```
Comment (namhong2001) [inline en/docs/cli/global.md]: `yarn global list` is not valid yet. `yarn global ls` command remained. v0.18.1

**Comment by gsklee** [comment]
If changes haven't gone live then the docs shouldn't be modified ahead of it. Right now the docs is in need of some adjustment to make it look right: https://yarnpkg.com/en/docs/cli/ls

**Comment by Haroenv** [comment]
Thanks for helping. If I see this correctly, this has been fixed by #377

**Pull request closed without merging**